specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/preview/2020-07-01-preview/examples/AzureADMetricsDelete.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "rg1",
+    "azureADMetricsName": "ddb1",
+    "api-version": "2020-07-01-preview",
+    "azureADMetricsConfig": {
+      "name": "myOrgAzureADMetrics",
+      "ownerTenantId": "950f8bca-bf4d-4a41-ad10-034e792a243d",
+      "resourceName": "myOrgVnetAzureADMetrics",
+      "subscriptionId": "57849194-ea1f-470b-abda-d195b25634c1",
+      "resourceGroup": "myOrgVnetRG"
+    }
```

Comment:
delete doesn't take parameters

Reply:
how does the deleting work if there is no input on which one to delete? i apologize if i am misinterpreting your comment.

Reply:
After I addressed your changes, I am getting errors about missing required property: location which I have not gotten before, as well as additional validation errors for the examples. Could you help me figure these errors out?